src/content/plugins/html-webpack-plugin.md

```diff
@@ -55,7 +55,7 @@ If you have multiple webpack entry points, they will all be included with `scrip
 tags in the generated HTML.
 
 If you have any CSS assets in webpack's output (for example, CSS extracted
-with the [ExtractTextPlugin](/plugins/extract-text-webpack-plugin))
+with the [MiniCssExtractPlugin](/plugins/mini-css-extract-plugin))
```

Comment:
Please add trailing slash to the URL, our dev server setup fails to open without it.

Reply:
you mean it should be corrected like this ?
 ```suggestion
with the [MiniCssExtractPlugin](/plugins/mini-css-extract-plugin/))
```

Reply:
yes please

Reply:
I have added